imports/client/components/App.tsx

```diff
@@ -319,12 +324,15 @@ const App = ({ children }: { children: React.ReactNode }) => {
       </ReactErrorBoundary>
     );
   }
+
+  const onPuzzlePage = useParams<"puzzleId">().puzzleId !== undefined;
```

Comment:
I don't really love this? It feels fragile and only tangentially connected to the question we're actually trying to answer. (What if we add another page with a puzzleId?)

On the other hand, I'm not sure we need it? I see two issues:

First, the border renders under the sidebar:

<img width="513" alt="Screenshot 2024-01-11 at 9 19 26 AM" src="https://github.com/deathandmayhem/jolly-roger/assets/28167/617dc41e-a47e-481f-9d85-13ede7b4a8e2">

I think this is just a fencepost error.

And second, the navbar is still scrollable even when the rest of the page content isn't:

https://github.com/deathandmayhem/jolly-roger/assets/28167/a4e73595-c92f-4025-846a-6c8338425ee2

And I think through mild shenanigans we can fix both of these in the `FixedLayout` component:

```TSX
const FixedLayoutDiv = styled.div`
  position: fixed;
  inset: /* top right bottom left */ calc(
      env(safe-area-inset-top, 0) + ${NavBarHeight} + 1px
    )
    env(safe-area-inset-right, 0) 0 env(safe-area-inset-left, 0);
`;

const FixedLayoutGlobal = createGlobalStyle`
  body {
    overscroll-behavior: none;
  }
`;

const FixedLayout = React.forwardRef<
  HTMLDivElement,
  React.HTMLAttributes<HTMLDivElement>
>((props, ref) => {
  return (
    <>
      <FixedLayoutGlobal />
      <FixedLayoutDiv ref={ref} {...props} />
    </>
  );
});

export default FixedLayout;
```

Reply:
Ah, neat -- if we do the `FixedLayout` shenanigans with disabling `overscroll-behavior`, we can get away with just always using the non-fixed navbar, even though it's the only part of the body whose content contributes to the flow (vs having fixed position)?  That works for me!